React/Profiler/RCTProfile.m

```diff
@@ -42,8 +42,7 @@
 
 #pragma mark - Variables
 
-// This is actually a BOOL - but has to be compatible with OSAtomic
-static volatile uint32_t RCTProfileProfiling;
+static atomic_bool RCTProfileProfiling = ATOMIC_VAR_INIT(NO);
```

Comment:
Hmm turns out we can't use `constexpr` because it implies `const` and we need to mutate this value.